thegreenweb/manifest.json

```diff
@@ -3,20 +3,26 @@
   "update_url":"http://clients2.google.com/service/update2/crx",
   "name": "The Green Web",
   "short_name": "The Green Web",
-  "version": "2.3.0",
+  "version": "2.4.0",
   "description": "Automatically check the sustainability of a website with The Green Web add-on.",
   "background": {
     "scripts": ["background.js","thegreenweb-utils.js"],
     "persistent": false
   },
   "minimum_chrome_version" : "22",
   "options_page": "options.html",
```

Comment:
are options_page and options_ui allowed to be there both? 

You tested this on chrome as well?

Reply:
Yes it turns out the options_page key is kinda deprecated:

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_page

But not entirely. options_ui, with the nested key of `open_in_tab: false`, is apparently what we ought to be using:

```js
"options_ui": {
    "page": "options.html",
    "open_in_tab": false
  },
```

Wow, what a mess!

Reply:
Yes, I have tested it on chrome - I had them both open when developing